app/src/main/res/layout/layout_appcoins_app_bar.xml

```diff
@@ -0,0 +1,50 @@
+<com.google.android.material.appbar.AppBarLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/app_bar"
+    android:layout_width="match_parent"
+    android:layout_height="@dimen/app_bar_height"
+    android:theme="@style/AppTheme.AppBarOverlay"
+    >
+
+  <com.google.android.material.appbar.CollapsingToolbarLayout
+      android:id="@+id/toolbar_layout"
+      android:layout_width="match_parent"
+      android:layout_height="match_parent"
+      android:background="@drawable/appbar_background_color"
+      android:fitsSystemWindows="false"
+      app:collapsedTitleGravity="start|center_vertical"
+      app:collapsedTitleTextAppearance="@style/ToolbarTextAppearance.Title"
+      app:expandedTitleGravity="center"
+      app:expandedTitleTextAppearance="@style/ToolbarTextAppearance.Title.Expanded"
+      app:layout_scrollFlags="scroll|exitUntilCollapsed"
+      app:toolbarId="@+id/toolbar"
+      >
+    <androidx.appcompat.widget.Toolbar
+        android:id="@+id/toolbar"
+        android:layout_width="match_parent"
+        android:layout_height="?android:attr/actionBarSize"
+        android:fitsSystemWindows="false"
+        app:layout_collapseMode="pin"
+        app:popupTheme="@style/AppTheme.PopupOverlay"
+        >
+      <TextView
+          android:id="@+id/transaction_timestamp"
+          style="@style/TransactionDetailsTextStyle.AppBar"
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
+          android:layout_gravity="end"
+          android:layout_marginTop="20dp"
+          android:layout_marginEnd="16dp"
+          android:visibility="gone"
```

Comment:
The view can be visible since in the setup UI we set it to Visible right?

Reply:
This was done to avoid misusage of this layout in other views (like it was being done before). It can be deleted now

Reply:
However I would let that there in case someone incorrectly includes this layout in another layout